db/payment_icons.yml

```diff
@@ -862,6 +862,10 @@
   name: mb
   label: Multibanco
   group: other
+-
+  name: multibanco
+  label: Multibanco
+  group: other
```

Comment:
What's happening to the old multibanco icon? Found it [here](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/mb.svg)